projects/rocr-runtime/runtime/hsa-runtime/core/runtime/amd_gpu_agent.cpp

```diff
@@ -1713,6 +1713,14 @@ hsa_status_t GpuAgent::GetInfo(hsa_agent_info_t attribute, void* value) const {
     case HSA_AMD_AGENT_INFO_PM4_EMULATION:
       *((bool*)value) = properties_.Capability2.ui32.AqlEmulationPm4_;
       break;
+    case HSA_AMD_AGENT_INFO_LUID:
+#ifdef __linux__
+      static_cast<hsa_luid_t*>(value)->low = 0;
+      static_cast<hsa_luid_t*>(value)->high = 0;
+#else
+      static_cast<hsa_luid_t*>(value)->low = properties_.LuidLowPart;
```

Comment:
Submit the interface changes together and remove #ifdef. The interface must not include any if/defs also. Linux thunk can keep them as 0.